.github/workflows/release.yml

```diff
@@ -116,14 +117,16 @@ jobs:
         working-directory: ./openvscode-server
         run: |
           set -e
-          VSCODE_REMOTE_SERVER_PATH="$GITHUB_WORKSPACE/vscode-reh-web-linux-x64" \
+          VSCODE_REMOTE_SERVER_PATH="$GITHUB_WORKSPACE/vscode-reh-web-linux-${{ matrix.arch }}" \
           ./resources/server/test/test-web-integration.sh --browser chromium
 
       - name: Run smoke tests
+        # Smoke tests requires a runner with the same architecture as the built vscode
+        if: matrix.arch == 'x64'
```

Comment:
I'm kinda curious what's failing in the smoke tests so they don't run for archs other than `x64` and the integrations tests can run in all archs.

Reply:
I also got curious. If you scroll down in the list of runs in my fork you will see that they fail without any apparent reason. All the tests fails one by one with timeout.

If you want I can try to investigate, but I thought it would be reasonable safe to skip the tests as they do run for one arch already.

Reply:
Took a look at the action runs in your fork and actually the integration test don't work either :sweat_smile: 
They fail because playwright is not supported for ARM https://github.com/microsoft/playwright/issues/2838

You should add this same check to skip integration tests for archs other than x64. Looking at the vscode ci actions, they do the same check.

Reply:
Cool. Will do. I will also check why the failures are being masked (they show as green).

Reply:
Sorry, but where do you see errors?

![image](https://user-images.githubusercontent.com/29582865/140858901-51c49361-8eff-40db-8df5-b3c7eaeb0196.png)

https://github.com/felipecrs/openvscode-releases/runs/4146273687?check_suite_focus=true#step:15:1

Reply:
Ok, so it seems that the test executor already skips it, and that's why it shows as green. But I guess it's more readable a blank skipped circle in GitHub Actions than this, so I did it.

Reply:
For integration test there aren't explicit errors like in the smoke test (guess there's some logic that just ignore them) but the tests are not run. If you check the integration test log output in the `x64` job you'll see additional logs for each test that successfully run

Reply:
Thank you for the explanation